src/main/java/com/dev/cinema/service/mapper/MovieMapper.java

```diff
@@ -0,0 +1,30 @@
+package com.dev.cinema.service.mapper;
+
+import com.dev.cinema.model.Movie;
+import com.dev.cinema.model.dto.movie.MovieRequestDto;
+import com.dev.cinema.model.dto.movie.MovieResponseDto;
+import com.dev.cinema.service.MovieService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieMapper {
+    private final MovieService movieService;
+
```

Comment:
do you use movie service somewhere in the MovieMapper?

Reply:
thanks, removing done